Makefile

```diff
@@ -14,38 +14,23 @@
 # `clean` run clean on esptool2, rboot, and rboot-sampleproject with inherited
 #				 configuration
 
-ESPTOOL2      ?= $(abspath esptool2/esptool2.exe)
-SDK_BASE      ?= C:/esp_iot_sdk_v1.1.2
-XTENSA_BINDIR ?= C:/xtensa-lx106-elf/bin

```

Comment:
O and you also don't need to use addprefix. So the line could be something like:
 CC := $(XTENSA_BINDIR)xtensa-lx106-elf-gcc 

Without the XTENSA_BINDIR set nothing will be prefixed and in your case it will. Of course the contents of XTENSA_BINDIR has to end in a slash for it to fit.